app/components/multiple_checked_select_component/multiple_checked_select_component.html.haml

```diff
@@ -0,0 +1,16 @@
+.ofn-drop-down.ofn-drop-down-v2{ data: { controller: "multiple-checked-select", "multiple-checked-select-input-name-value": @name } }
+  %div.ofn-drop-down-label{ "data-multiple-checked-select-target": "button" }
+    %span{class: "label"}= t('admin.columns')
+    %span{ class: "icon-caret-down", "data-multiple-checked-select-target": "caret" }
+  %div.menu{ class: "hidden", "data-multiple-checked-select-target": "options" }
+    %div.filter
+      %input{ type: "text", "data-multiple-checked-select-target": "filter", placeholder: @filter_placeholder }
```

Comment:
Why is `@filter_placeholder` a variable? Can't it just be a translation inserted here?

Reply:
Yep, I agree: this don't need to be a variable (at least for now). We'll change that in a future if it's necessary.

Will do the changes.

Reply:
See commit 3ec740798e